### PR TITLE
Merge 'tensorboard-integration-ku' with 'tensorboard-integration'

### PR DESCRIPTION
**Pull request by kadhirumasankar**
Added a function to `logger.py` which logs metrics (namely model score, the best model score, EMA score, and the best EMA score) to TensorBoard files inside `~/runs/uav-classifier/`

TensorBoard can be run using `tensorboard --logdir=~/runs/uav-classifier/`

### CODE REVIEW

File: train/train_utils/logger.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import pathlib
 import logging
+from torch.utils.tensorboard import SummaryWriter
```
Comment (alexwitt23) [inline train/train_utils/logger.py]: See [here](https://google.github.io/styleguide/pyguide.html#22-imports) on imports. Only import packages and modules.

File: train/train_utils/logger.py [hunk 2]
```diff
@@ -35,4 +37,6 @@ def warning(self, message: str) -> None:
     def error(self, message: str) -> None:
         logging.error(message)
 
-    # def metric
+    # logs metric to tensorboard
+    def metric(self, tag, value, epoch):
```
Comment (alexwitt23) [inline train/train_utils/logger.py]: `def metric(self, tag: str, value: float, epoch: int) -> None:`

**Review by alexwitt23 — APPROVED** [review]
lgtm
